src/js/pages/Escrow/index.jsx

```diff
@@ -150,7 +151,9 @@ class Escrow extends Component {
             <Offer offer={offer} prices={prices}/>
           </Col>
         </Row>
-        <OpenChat statusContactCode={offer.user.statusContactCode} />
+        <OpenChat statusContactCode={offer.user.statusContactCode} withBuyer={!isBuyer} />
+        <Profile withBuyer={!isBuyer} address={isBuyer ? escrow.offer.owner : escrow.buyer} />
+        <hr />
```

Comment:
It's better to use borders if possible

Reply:
I think in this case it's okay because in the design there's a margin-top|bottom between the HR and the elements.